lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1619,6 +1610,12 @@ public function get_item_schema() {
 				'type'        => 'boolean',
 				'context'     => array( 'view', 'edit' ),
 			);
+
+			$schema['properties']['password'] = array(
+				'description' => __( 'A password to protect access to the content and excerpt.' ),
+				'type'        => 'string',
+				'context'     => array( 'edit' ),
+			);
```

Comment:
I think I answered my question here... we can't have `.password` at the top level because that's where the string that sets that password must live. But I'm still 👎  on embedding the `.protected` key within the content and excerpt blocks; I'd prefer one of,

1. Have a `.password` property on the top level that is set as a string, and can be read as a string when password-authenticated, but otherwise displays as the boolean `true`; or
2. Have a `.protected` boolean property on the top level of the post object that represents whether the string `.password` field is present (whether a password is required)